test/unit/test.redispatch.js

```diff
@@ -33,6 +33,20 @@ QUnit.module('Redispatch', {
 
         stitchedAds() {
           return false;
+        },
+
+        isResumingAfterNoPreroll() {
+          return false;
+        },
+
+        _state: {
```

Comment:
Why is this needed now?

Reply:
🤔 I might be able to remove this now actually